luxonis_train/callbacks/gpu_stats_monitor.py

```diff
@@ -0,0 +1,287 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
I know lightning is currently a requirements anywhere, but should we explicitly mention this in the README as well?
Is there a known reason why this was removed in lightning 2 btw?

Reply:
>I know lightning is currently a requirements anywhere, but should we explicitly mention this in the README as well?

Yeah we can mention that `luxonis-train` is build on top of lightning. 

>Is there a known reason why this was removed in lightning 2 btw?

It wasn't removed entirely; it was replaced by the `DeviceStatsMonitor`, but the `DeviceStatsMonitor` doesn't use `nvidia-smi`, only torch memory stats, which don't include the overall gpu utilization. There's some discussion about it in this [issue](https://github.com/Lightning-AI/pytorch-lightning/issues/11796), but there's been no new activity since 2022.

Reply:
> Yeah we can mention that luxonis-train is build on top of lightning.

Ok, let's mention it. Sounds good on the rest.